Package.swift

```diff
@@ -26,11 +26,14 @@ func resolveTargets() -> [Target] {
                 sources: ["Purchases", "Purchases/Public"],
                 publicHeadersPath: "Purchases/Public",
                 cSettings: [
-                    .headerSearchPath("Purchases"),
-                    .headerSearchPath("Purchases/Public"),
-                    .headerSearchPath("Purchases/Caching"),
-                    .headerSearchPath("Purchases/SubscriberAttributes")
-                ])]
+                    .headerSearchPath("Caching"),
```

Comment:
this new structure has the side-effect of removing the protected group from header search paths for CocoaPods and SPM. 
it means that if you're using protected stuff, it'll crash. that's probably a good thing, but I'm not 100% sure if any of the hybrid ones would break because of it. That's a part of why I put this on hold. We could always just add the lines here to keep the same behavior as before, but I figured it might be a nice moment to actually think about what the best path is

Reply:
I just checked and it looks like the hybrid sdks don't use the protected stuff (which is how it should be, but still worth checking)

Reply:
nevermind, these are header search paths, I don't know what I was thinking, protected should be included here. Just re-added.